packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -29,7 +30,10 @@ export class ExpressionParser {
   getVariables(expression) {
     this.validate(expression);
     const nodeTree = this.math.parse(expression);
-    return nodeTree.filter(node => node.isSymbolNode).map(({ name }) => name);
+    const result = nodeTree
+      .filter(node => node.isSymbolNode && !Object.keys(this.customFunctions).includes(node.name))
```

Comment:
For some reasons mathjs just treat the custom function as a symbol node so this is just to exclude that when we extract variables from the formula